src/main/java/pinball/PinballComponent.java

```diff
@@ -1,16 +1,187 @@
 package pinball;
 
+import com.badlogic.gdx.math.MathUtils;
 import com.badlogic.gdx.math.Vector2;
-import com.badlogic.gdx.physics.box2d.World;
+import com.badlogic.gdx.physics.box2d.*;
 
 import javax.swing.JComponent;
+import java.awt.*;
 
 public class PinballComponent extends JComponent {
 
+    private long time;
+
+    private static final float WIDTH = 1000;
+    private static final float HEIGHT = 1500;
+
+    private static final float CORNER_LENGTH = 175;
+    private static final float BASE_LENGTH = 350;
+    private static final float FLIPPER_LENGTH = 120;
+
+    private static final float BOX_TO_SCREEN = 10f;
+    private static final float SCREEN_TO_BOX = 1f / BOX_TO_SCREEN;
+
+    private final int radius = 30;
+
     private final World world;
+    private final Body bottom, right, left, top, divider;
+    private final Body topRightCorner, topLeftCorner, bottomRightBase, bottomLeftBase;
+    private final Body rightFlipper, leftFlipper;
+    private final Body ball;
 
-    public PinballComponent() {
+    PinballComponent()
+    {
         world = new World(new Vector2(0, 9.8f), false);
+
+        //set up all horizontal and vertical lines
+        bottom = createWall(100f * SCREEN_TO_BOX, (HEIGHT + 100) * SCREEN_TO_BOX, WIDTH * SCREEN_TO_BOX, 1f * SCREEN_TO_BOX);
+        top = createWall(100f * SCREEN_TO_BOX, 100f * SCREEN_TO_BOX, WIDTH * SCREEN_TO_BOX, 1f * SCREEN_TO_BOX);
+        left = createWall(100f * SCREEN_TO_BOX, 100f * SCREEN_TO_BOX, 1f * SCREEN_TO_BOX, HEIGHT * SCREEN_TO_BOX);
+        right = createWall((WIDTH + 100) * SCREEN_TO_BOX, 100f * SCREEN_TO_BOX, 1f * SCREEN_TO_BOX, HEIGHT * SCREEN_TO_BOX) ;
+        divider = createWall(1025 * SCREEN_TO_BOX, 300f * SCREEN_TO_BOX, 1f * SCREEN_TO_BOX, (HEIGHT - 200f) * SCREEN_TO_BOX);
+
+        //set up all diagonal lines
+        topRightCorner = createDiagonalLine(950f * SCREEN_TO_BOX, 100f * SCREEN_TO_BOX, CORNER_LENGTH * SCREEN_TO_BOX, 30);
+        topLeftCorner = createDiagonalLine(250f * SCREEN_TO_BOX, 100f * SCREEN_TO_BOX, CORNER_LENGTH * SCREEN_TO_BOX, 150);
+        bottomRightBase = createDiagonalLine(1025 * SCREEN_TO_BOX, 1230f * SCREEN_TO_BOX, BASE_LENGTH * SCREEN_TO_BOX, 145);
+        bottomLeftBase = createDiagonalLine(100f * SCREEN_TO_BOX, 1230f * SCREEN_TO_BOX, BASE_LENGTH * SCREEN_TO_BOX, 35);
+
+        //set up flippers (static for now, but soon to be dynamic and jointed)
+        rightFlipper = createDiagonalLine(735 * SCREEN_TO_BOX, 1435 * SCREEN_TO_BOX, FLIPPER_LENGTH * SCREEN_TO_BOX, 145);
+        leftFlipper = createDiagonalLine(390 * SCREEN_TO_BOX, 1435 * SCREEN_TO_BOX, FLIPPER_LENGTH * SCREEN_TO_BOX, 35);
+
+        //cue the ball
+        ball = createBall(1060 * SCREEN_TO_BOX, 1565 * SCREEN_TO_BOX, radius * SCREEN_TO_BOX);
```

Comment:
Move the SCREEN_TO_BOX multiplication into the create() methods to make this easier to read.

Reply:
This will be fixed in Kaila's PR.